internal/pkg/cli/svc_deploy.go

```diff
@@ -444,6 +457,20 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 	return nil
 }
 
+func (o *deploySvcOpts) validateAppVersion(svc *manifest.LoadBalancedWebService) error {
+	appVersion, err := o.appVersionGetter.Version()
+	if err != nil {
+		return fmt.Errorf("get version for app %s: %w", o.appName, err)
+	}
+	if aws.StringValue(svc.Alias) != "" && o.targetApp.RequiresDNSDelegation() {
+		diff := semver.Compare(appVersion, deploy.AliasLeastAppTemplateVersion)
+		if diff < 0 {
+			return fmt.Errorf("enabling https alias requires the application version to be at least %s", deploy.AliasLeastAppTemplateVersion)
```

Comment:
💭  I sort of get the impression that our error message is more of a description of the situation that causes the error - something like `"the application version is below 1.0.0 (the least version of template that enables alias)"` - as opposed to instructional (a statement that describes the requirement). 💭